### PR TITLE
Update webcam module to set the camera framerate

### PR DESCRIPTION
**Pull request by f-fl0**
#### Description

This PR updates the [webcam](https://github.com/blackjack/webcam ) module to `ba064708e1652773bdb71e0605a1819ef4fad2e7` which allows to set (and get) the video camera capture framerate based on the `FrameRate` value specified in `prop.Media`.

### CODE REVIEW

File: pkg/driver/camera/camera_linux.go [hunk 1]
```diff
@@ -179,6 +179,11 @@ func (c *camera) VideoRecord(p prop.Media) (video.Reader, error) {
 		return nil, err
 	}
 
+	err = c.cam.SetFramerate(float32(p.FrameRate))
```
Comment (at-wat) [inline pkg/driver/camera/camera_linux.go:182]: How it behaves when user doesn't explicitly specify frame rate?

Comment (f-fl0) [reply pkg/driver/camera/camera_linux.go:182]: Good point. I will check that.

**Comment by f-fl0** [comment]
`15973f6961cbe56d44372d74f46abe88598265a1` failed in CI because of the following error:
```
=== RUN   TestThrottle
    throttle_test.go:36: Number of pushed images is expected to be 20, but pushed 27
--- FAIL: TestThrottle (0.54s)
FAIL
```

It seems irrelevant to the changes introduced in this PR.

File: pkg/driver/camera/camera_linux.go (hunk 1 above)
Comment (f-fl0) [reply pkg/driver/camera/camera_linux.go:182]: I added some tests. When the framerate is not specified the constraint seems to be ignored.

Comment (at-wat) [reply pkg/driver/camera/camera_linux.go:182]: `p.FrameRate` seems to be 0 when user doesn't set frame rate.
I think it's better not to call SetFramerate in this case.

**Review by at-wat — APPROVED** [review]
LGTM

**Pull request merged**